lib/ex_doc/formatter/html/autolink.ex

```diff
@@ -596,20 +608,29 @@ defmodule ExDoc.Formatter.HTML.Autolink do
       |> String.split(" ")
       |> Enum.split(-1)
 
-    {"", Enum.join(mod, "."), hd(name), arity}
+    {"", Enum.join(mod, "."), hd(name), split_arities(arity)}
   end
 
   # handles "/" function
   defp split_function_list([modules, "", arity]) when is_binary(modules) do
     split_function_list([modules <> "/", arity])
   end
 
+  defp split_arities(arities) do
+    if String.contains?(arities, ","), do: String.split(arities, ","), else: arities
```

Comment:
Doesn't this mean someitmes it is a list and sometimes it is a string? If so,  I would prefer to always make it a non-empty list. It would require less conditionals in the code.

Reply:
It does and you're absolutely right. I am working on changing this, since it would also cause a lot more unnecessary code when supporting the same for built-in types/functions/callbacks.

Reply:
Changed `split_arities/1` to always return a list.    
Consuming functions will use helpers like `first_arity_of/1` now.